src/app/mock-question.ts

```diff
@@ -0,0 +1,5 @@
+export const Questions: Object[] = [
```

Comment:
Please change Object to special interface that will describe Question entity

Reply:
was applied